### ISSUE TITLE
diffrence in colors presentation between vim/gvim

### ISSUE THREAD

**Issue by r8b7xy** [lilydjwg/colorizer#22]
Hex colors in my Xdefaults look diffrently when viewing them in vim and gvim.
Please look at S_base03/02.
![bug](http://i.imgur.com/PEqFM8J.png)
Is it known bug?

**Comment by lilydjwg** [comment]
It's a known limitation. Most terminals only support 256 colors, so colorizer has to "round up" the 256^3 colors to those, sometimes it will give poor results. Better results will require more time (and code) to compute.

**Comment by r8b7xy** [comment]
Is this something east to fix or rather something to accept? :)

**Comment by lilydjwg** [comment]
No. You need a terminal that supports more colors in the first place, which I've never seen.

**Comment by r8b7xy** [comment]
Here is the list of true color terminals: [Colors in terminal](https://gist.github.com/XVilka/8346728)
As far as I see it will be better to wait for true color support in urxvt :)

**Comment by lilydjwg** [comment]
Wow, that's great! But then we'll wait for Vim to support it. I see ZyX's branch. I'll try that when I have time :-)
